test/lib/compile/csdl2openapi.test.js

```diff
@@ -103,6 +103,22 @@ describe('Edge cases', function () {
         assert.deepStrictEqual(openapi, expected, 'Empty CSDL document');
     })
 
+    it('tags object exists and it does not have duplicates', function () {
+        const csdl = {
+            $EntityContainer: 'this.Container',
+            this: {
+                ET: { $Kind: 'EntityType', $Key: ['key'], key
+                : {} },
+                Container: {
+                    Set: { $Type: 'this.ET', $Collection: true }
+                }
+            }
+        };
+        const openapi = lib.csdl2openapi(csdl, {});
+        assert.ok(openapi.tags, 'Tags object exists');
```

Comment:
How are you providing duplicate values in this sample?

Reply:
made changes